vtex-sites/storecomponents.store
# Add and conofigure @vtex/gatsby-plugin-onesignal

## What's the purpose of this pull request?

Add newly created `@vtex/gatsby-plugin-onesignal` and set it up so that this store supports OneSignal notifications.

## How it works? 

This PR should provide more context: https://github.com/vtex/faststore/pull/663.

## How to test it?

Go to the latest deploy preview linked below and navigate for a few seconds until you see a banner asking if you want to receive push notifications. If you allow it, everything should be working and you're now able to receive notifications from OneSignal.
